aws/extensions/computetask_linux_hamletenv/extension.ftl

```diff
@@ -0,0 +1,85 @@
+[#ftl]
+
+[@addExtension
+    id="computetask_linux_hamletenv"
+    aliases=[
+        "_computetask_linux_hamletenv"
+    ]
+    description=[
+        "Uses the shared profile configuration to set default environment varaibles"
+    ]
+    supportedTypes=[
+        EC2_COMPONENT_TYPE,
+        ECS_COMPONENT_TYPE,
+        COMPUTECLUSTER_COMPONENT_TYPE,
+        BASTION_COMPONENT_TYPE
+    ]
+    scopes=[
+        COMPUTETASK_EXTENSION_SCOPE
+    ]
+/]
+
+[#macro shared_extension_computetask_linux_hamletenv_deployment_computetask occurrence ]
+
+    [#local baselineLinks = _context.BaselineLinks]
+    [#local baselineComponentIds = getBaselineComponentIds(baselineLinks)]
+    [#local operationsBucket = getExistingReference(baselineComponentIds["OpsData"]) ]
+    [#local dataBucket = getExistingReference(baselineComponentIds["AppData"])]
+    [#local envVariables = getFinalEnvironment(occurrence, _context ).Environment ]
+
+    [#local role = (occurrence.Configuration.Settings.Product["Role"].Value)!""]
+
+    [#local envContent = [
+        r'# Set environment variables from hamlet configuration',
+        r'export cot_request="'       + getCLORequestReference()        + '"',
```

Comment:
Time to switch to `hamlet_*` from `cot_*`?

Reply:
Done, I don't think these ones are actually used anymore, but didn't want to change to much of the functionality while also moving everything around